### ISSUE TITLE
[Bug]: After scrolling on the widget, there’s ghosting until it refocuses. This is definitely an Electron issue, not CSS—I confirmed this because it doesn’t happen on the localhost:5173 page.

### ISSUE THREAD

**Issue by InfiniteLoopAlchemist** [electron/electron#43496]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

32.0.1

### What operating system(s) are you using?

macOS

### Operating System Version

MacOS Sonoma Version 14.6.1 (23G93)

### What arch are you using?

arm64 (including Apple Silicon)

### Expected Behavior

The desktop widget renders just like it does on a transparent webpage, and while scrolling, the text doesn’t exhibit any ghosting.
![Image 8-28-24 at 16 29](https://github.com/user-attachments/assets/4320a560-16bd-486e-beeb-aa30e5554d09)

### Actual Behavior

After scrolling on the widget, there’s noticeable ghosting until it comes back into focus. This is definitely an Electron issue, not related to CSS—I’ve confirmed that. Also, I’m seeing ghosting intermittently when I unfocus and quickly refocus the widget. I recorded a video showing the behavior.
![Image 8-28-24 at 12 52](https://github.com/user-attachments/assets/3a9c24f9-b55f-4442-86e2-1fc1a82ce7cf)
![Image 8-28-24 at 16 19](https://github.com/user-attachments/assets/f05d1f4e-db28-40be-9784-46f09fc89f47)

https://youtube.com/shorts/ZJ0d3ebB_QU?feature=share

### Testcase Gist URL

[Ghosting Bug](https://gist.github.com/InfiniteLoopAlchemist/5b6babd072fdd8dce9ffa450fd46cf8d)

### Additional Information

from main.mjs
const createWindow = () => {
    const mainWindow = new BrowserWindow({
        width: 500,
        height: 800,
        frame: false,
        transparent: true,
        alwaysOnTop: false,
        resizable: false,
        fullscreenable: false,
        skipTaskbar: true,
        webPreferences: {
            preload: join(__dirname, 'preload.mjs'),
            nodeIntegration: false,
            contextIsolation: true
        }
    });
    mainWindow.setPosition(100, 100);

from package.json
  "devDependencies": {
    "@electron-forge/cli": "^7.4.0",
    "@electron-forge/maker-deb": "^7.4.0",
    "@electron-forge/maker-rpm": "^7.4.0",
    "@electron-forge/maker-squirrel": "^7.4.0",
    "@electron-forge/maker-zip": "^7.4.0",
    "@electron-forge/plugin-auto-unpack-natives": "^7.4.0",
    "@electron-forge/plugin-fuses": "^7.4.0",
    "@electron-forge/plugin-vite": "^7.4.0",
    "@electron/fuses": "^1.8.0",
    "autoprefixer": "^10.4.20",
    "electron": "32.0.1",
    "postcss": "^8.4.41",
    "tailwindcss": "^3.4.10",
    "vite": "^5.0.12"
  },
  "keywords": [],
  "author": {

  },
  "license": "MIT",
  "packageManager": "yarn@1.22.22+<removed just incase>",
  "dependencies": {
    "axios": "^1.7.5",
    "electron-squirrel-startup": "^1.0.1",
    "moment": "^2.30.1",
    "moment-timezone": "^0.5.45",
    "react": "^18.3.1",
    "react-dom": "^18.3.1",
    "sunrise-sunset-js": "^2.2.1"

I have not tested on linux or windows yet.

**Comment by nikwen** [comment]
If you provide code to reproduce this (e.g. via [Electron Fiddle](https://www.electronjs.org/fiddle)), it'll be easier for Electron maintainers to fix the issue.

**Comment by InfiniteLoopAlchemist** [comment]
> If you provide code to reproduce this (e.g. via [Electron Fiddle](https://www.electronjs.org/fiddle)), it'll be easier for Electron maintainers to fix the issue.

Will do tomorrow.

**Comment by InfiniteLoopAlchemist** [comment]
> If you provide code to reproduce this (e.g. via [Electron Fiddle](https://www.electronjs.org/fiddle)), it'll be easier for Electron maintainers to fix the issue.

https://gist.github.com/InfiniteLoopAlchemist/b8b9897d8f524cff2795abb29b711e25

**Comment by nikwen** [comment]
Yes, it's the changing background! When I place a YouTube video below the window, I get that "ghosting" behavior, too.

<img width="506" alt="Screenshot" src="https://github.com/user-attachments/assets/bb6bc954-23fa-4e12-93ad-f92bb34e18c2">

**Comment by InfiniteLoopAlchemist** [comment]
> Yes, it's the changing background! When I place a YouTube video below the window, I get that "ghosting" behavior, too.
> 
> <img alt="Screenshot" width="506" src="https://private-user-images.githubusercontent.com/3309657/362787275-bb6bc954-23fa-4e12-93ad-f92bb34e18c2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ5NTYzOTcsIm5iZiI6MTcyNDk1NjA5NywicGF0aCI6Ii8zMzA5NjU3LzM2Mjc4NzI3NS1iYjZiYzk1NC0yM2ZhLTRlMTItOTNhZC1mOTJiYjM0ZTE4YzIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MjlUMTgyODE3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Y2I5ODQ4Y2ZlMjk5NjdjNjM4NTlkN2Y2ZjQwYWNkNmQ1MDZiOWZiMzYyMDIwYTQ0YTc1NTE0NTJkYmEwNzVkYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.1S5v5ykSKRlNcpDBWft1w_pKmoz5_wl8Qr01NWVnmCY">

Yea I normally use 15 min between new desktop images. And it happens when using every 5 sec and 5 min

**Comment by InfiniteLoopAlchemist** [comment]
![Image 9-5-24 at 13 18](https://github.com/user-attachments/assets/884bc0c1-63da-42c9-af8e-d23877e382cb)
![Image 9-5-24 at 13 19](https://github.com/user-attachments/assets/2c82d23e-131d-461f-a294-e3205be52052)
Also notice if you use tailwind fixed property with bottom-0 the background not fill while scrolling/
